src/react-vite/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -200,6 +200,8 @@ code changes to accomplish the following:
   </figcaption>
 </figure>
 
+<strong>Having issues with your local setup?</strong> You can use either [StackBlitz](https://stackblitz.com/fork/github/bitovi/academy/tree/main/exercises/react-vite/02-setting-up-your-environment/problem?file=src/App.tsx) or [CodeSandbox](https://codesandbox.io/p/devbox/github/bitovi/academy/tree/main/exercises/react-vite/02-setting-up-your-environment/problem?file=src/App.tsx) to do this exercise in an online code editor.
+
```

Comment:
I meant to ask about how to structure these links yesterday, good to know the syntax. 👍